packages/trader/src/sass/app/_common/components/video-player.scss

```diff
@@ -0,0 +1,283 @@
+            }
+
+            &__filled {
+                position: relative;
+                width: 0%;
+                height: 0.4rem;
+                border-radius: $BORDER_RADIUS;
+                background-color: var(--brand-red-coral);
+
+                &--animated {
+                    transition: all 0.3s linear;
+                }
+
+                &--ended {
+                    width: 100%;
+                }
+            }
+        }
+
+        &__bottom-bar {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            height: 3.6rem;
+            padding-bottom: 0.4rem;
+
+            &--blocked {
+                pointer-events: none;
+            }
+
+            .controls__right {
+                width: 10.2rem;
+            }
+        }
+
+        &__time-wrapper {
+            margin: 0 1.6rem;
+        }
+
+        &__button {
+            background: none;
+            border: none;
+            padding: 0;
+            cursor: pointer;
+            width: 2.4rem;
+            height: 2.4rem;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+
+            svg {
+                margin: 0;
+            }
+        }
+    }
+
+    &__volume__wrapper {
+        position: relative;
+
+        &:before {
+            content: '';
+            display: block;
+            height: 1rem;
+            width: 4rem;
+            position: absolute;
+            right: -1rem;
+        }
+    }
+
+    &__volume-bar {
+        position: relative;
+        width: 0.4rem;
+        height: 100%;
+        border-radius: $BORDER_RADIUS;
+        cursor: pointer;
+        background-color: var(--text-less-prominent);
+
+        &__wrapper {
+            width: 4rem;
+            height: 10.4rem;
+            position: absolute;
+            bottom: 2.5rem;
+            right: -1rem;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            padding: 0.8rem 0;
+            border-radius: $BORDER_RADIUS * 2;
+            background-color: var(--general-main-2);
+            box-shadow: 0 0.8rem 1.6rem 0 var(--shadow-menu);
+            transform: translateY(0);
+            transition: transform 0.25s ease, opacity 0.25s linear;
+            opacity: 0;
+
+            &--enter,
+            &--exit {
+                transform: translateY(1rem);
+                opacity: 0;
+            }
+            &--enter-done {
+                transform: translateY(0);
+                opacity: 1;
+            }
+        }
+
+        &__filled {
+            position: absolute;
+            bottom: 0;
+            width: 0.4rem;
+            border-radius: $BORDER_RADIUS;
+            background-color: var(--brand-red-coral);
+
+            &--animated {
+                transition: all 0.3s linear;
+            }
+        }
+    }
+
+    &__progress-dot,
+    &__volume-dot {
```

Comment:
let's put all __volume-related classnames under &__volume :)

Reply:
Will move, but in this particular place I applied one style for 2 classnames (`&__progress-dot,&__volume-dot`). If remove `&__volume-dot` in `&__volume {...}`, then I need to duplicate this lines. Should I do it?